index.html

```diff
@@ -21,18 +23,203 @@
+            </p>
+            <h2 href="#" class="about-connect">LET'S CONNECT</h2>
+
+            <ul class="social-media-list">
+                <li class="social-media-item">
+                    <a href=""><img src="assets/github.svg" alt="Cristian's GitHub" class="social-icon"></a>
+                </li>
+                <li class="social-media-item">
+                    <a href=""><img src="assets/linkedin.svg" alt="Cristian's LinkedIn" class="social-icon"></a>
+                </li>
+                <li class="social-media-item">
+                    <a href=""><img src="assets/angel.svg" alt="Cristian's Angelist" class="social-icon"></a>
+                </li>
+                <li class="social-media-item"> 
+                    <a href=""><img src="assets/twitter.svg" alt="Cristian's Twitter" class="social-icon"></a>
+                </li>
+                <li class="social-media-item"> 
+                    <a href=""><img src="assets/medium.svg" alt="Cristian's Medium" class="social-icon"></a>
+                </li>
+            </ul>
+            <!-- Resume button -->
+            <button type="button" class="resume-button">Get my resume</button>
+        </div>
+        <!-- Skills Section -->
+        <ul>
+        <div class="skills-section">
+          <ul>
+            <ul class="skills-container">
+              <!-- Languages -->
+              <li class="skills-title">Languages</li>
+              <li class="skills-image-container">
+                  <figcaption >
+                      <img src="assets/down-arrow.svg" alt="down-arrow-vector" class="skills-image">
+                  </figcaption>
+              </li>
+          </ul>
+          <ul class="skills-list">
+              <li class="skills-item">
+                  <div class="language-item">
+                      <figcaption class="language-image-container">
+                          <img class="language-image" src="./assets/js.png" alt="javascript">
+                      </figcaption>
+                      <h2 class="skills-item-title">JavaScript</h2>
+                  </div>
+              </li>
+              <li class="skills-item">
+                  <div class="language-item">
+                      <figcaption class="language-image-container">
+                          <img src="./assets/html.png" alt="html" class="language-image">
+                      </figcaption>
+                      <h2 class="skills-item-title">HTML</h2>
+                  </div>
+              </li>
+              <li class="skills-item">
+                  <div class="language-item">
+                      <figcaption class="language-image-container">
+                          <img src="assets/css.png" alt="css" class="language-image">
+                      </figcaption>
+                      <h2 class="skills-item-title">CSS</h2>
+                  </div>
+              </li>
+          </ul>
```

Comment:
- [ ] As pointed out by your earlier reviewer, please make sure that all the technology categories (`"Languages"`, `"Frameworks"` and `"Skills"`) are in a `<ul>`, each of them in a `<li>` 🦭 

Kindly follow the guidelines below provided by the earlier reviewer 🎯 

![image](https://github.com/CVILLA09/Portfolio/assets/117144028/4292ade2-0816-437c-a4c7-caefe27d3235)

From the above, each technology section gets its own `<li>` and are all found inside one `<ul>`
The technologies can also make use of sub `<ul>'s` and `<li>'s` to make their own sub-lists 🪜 

Hence
**The technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`**